scripts/jenkins/cloud/ansible/install-crowbar.yml

```diff
@@ -41,6 +41,8 @@
             src: "{{ crowbar_batch_file }}"
             dest: "{{ admin_crowbar_batch_file }}"
 
+        - name: ERROR - Dark side want you to fail!!!
```

Comment:
:dark_sunglasses:

Reply:
yeah that was a placeholder to test gathering artifacts on a failed job. O:)